migrations/versions/rev-2021-01-13-12:50:33-8277175ec726_.py

```diff
@@ -0,0 +1,43 @@
+"""empty message
+
+Revision ID: 8277175ec726
+Revises: fc331e3f9322
+Create Date: 2021-01-13 12:50:33.660992
+
+"""
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+
+# revision identifiers, used by Alembic.
+revision = '8277175ec726'
+down_revision = 'fc331e3f9322'
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('groups',
+    sa.Column('deleted_at', sa.DateTime(timezone=True), nullable=True),
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(), nullable=False),
+    sa.Column('user_id', sa.Integer(), nullable=True),
+    sa.Column('created_at', sa.DateTime(timezone=True), nullable=True),
+    sa.Column('modified_at', sa.DateTime(timezone=True), nullable=True),
+    sa.ForeignKeyConstraint(['user_id'], ['users.id'], ondelete='CASCADE'),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.add_column('events', sa.Column('group_id', sa.Integer(), nullable=True))
+    op.create_foreign_key(None, 'events', 'groups', ['group_id'], ['id'], ondelete='SET NULL')
```

Comment:
Add name to the foreign key here, so it becomes possible to drop it below

See other migrations for reference